python/cmake/thirdparty/get_nvcomp.cmake

```diff
@@ -12,29 +12,22 @@
 # the License.
 # =============================================================================
 
+set(KVIKIO_USE_PROPRIETARY_BINARY ON)
```

Comment:
You might want to make this an option so a user can opt-out.

Reply:
Presently I've decided to only support 2.3.3 as I'd need to rewrite a bunch of tests and determine what is running in order to swap out their variations. If you don't mind I'd like to commit as is and consider that down the road.